### PR TITLE
Documented workaround for --rule-names auto-completion in troubleshooting, mentioned in command reference

### PR DESCRIPTION
**Pull request by stepgerner**
Based on https://jira.wandisco.com/browse/DOCU-982

### CODE REVIEW

File: docs/command-reference.md [hunk 1]
```diff
@@ -1913,6 +1913,10 @@ SYNOPSYS
 hive migration add --source sourceAgent --target remoteAgent --rule-names test_dbs,user_dbs --name hive_migration --auto-start
 ```
 
+:::note
```
Comment (lucasdigital) [inline docs/command-reference.md:1916]: This works! I'm not familiar with which admonitions we used. If we have a warning or tip admonition (text block for alerts), might that be more suitable?

**Review by lucasdigital — APPROVED** [review]
LGTM

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -61,3 +61,16 @@ And the following setting determines the low watermark percentage:
 ```text title="Example"
 notifications.pending.region.clear.percent=50
 ```
+
+### Rule names parameter does not autocomplete in the CLI
+
+When adding the `--rule-names` parameter to the end of a `migration add` command, auto-completion will not suggest the parameter name. For example:
+
+```text="Example"
+WANdisco LiveData Migrator >> hive migration add --name test --source sourceAgent --target testGlue --ru
```
Comment (benradstone) [inline docs/troubleshooting.md:70]: I know this is copied verbatim from the Engineering example, but include the full parameter at the end here:

`WANdisco LiveData Migrator >> hive migration add --name test --source sourceAgent --target testGlue --rule-names`

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -61,3 +61,16 @@ And the following setting determines the low watermark percentage:
 ```text title="Example"
 notifications.pending.region.clear.percent=50
 ```
+
+### Rule names parameter does not autocomplete in the CLI
+
+When adding the `--rule-names` parameter to the end of a `migration add` command, auto-completion will not suggest the parameter name. For example:
+
+```text="Example"
```
Comment (benradstone) [inline docs/troubleshooting.md:69]: This should read as:

```text title="Example"

Comment (stepgerner) [reply docs/troubleshooting.md:69]: Crap, well spotted. Thanks!

File: docs/troubleshooting.md (hunk 2 above)
Comment (stepgerner) [reply docs/troubleshooting.md:70]: Thank you! I wasn't sure.

**Pull request merged**